src/Application/Migrations/23_function_encoding_profile_all_dependee.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION encoding_profile_all_dependees(param_profile_id bigint, param_known_ids bigint[] DEFAULT array[]::bigint[])
+  RETURNS TABLE (id bigint) AS
+$$
+DECLARE
+  p bigint;
```

Comment:
"p" doesn't have an obvious meaning.
at least "p_id" would be nice or "dependee_profile_id" or similar